<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MooveWidget</class>
 <widget class="QWidget" name="MooveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="goForwardButton">
       <property name="text">
        <string>go forward</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goBackButton">
       <property name="text">
        <string>go back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="directionLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
